DataFormats/Parameters/include/DataFormatsParameters/AggregatedRunInfo.h

```diff
@@ -36,6 +38,7 @@ struct AggregatedRunInfo {
   int64_t orbitEOR;    // orbit when run ends after orbit reset
 
   // we may have pointers to actual data source objects GRPECS, ...
+  o2::parameters::GRPECSObject* grpECS = nullptr; // pointer to GRPECSobject (fetched during struct building)
```

Comment:
@sawenzel Could you make it `const o2::parameters::GRPECSObject* grpECS = nullptr;` ?

Reply:
yes of course